meteor/sofie-server/model/entities/basic-rundown.ts

```diff
@@ -0,0 +1,21 @@
+export class BasicRundown {
+	readonly id: string
+	readonly name: string
+	protected isRundownActive: boolean
+	protected lastTimeModified: number
+
+	constructor(id: string, name: string, isActive: boolean, lastTimeModified: number) {
```

Comment:
Starting to get a lot of parameters here. Not saying we should do anything about it right now, but we need to keep it in mind.

If we wanted we could make a `BasicRundownInterface` and give that as a parameter. Then would could extend `RundownInterface` with `RundownBasicInterface`.

Reply:
I have left it for now, but if more are added we should indeed make that change.